scripts/regenerate_piper_catalog.py

```
@@ -0,0 +1,357 @@
+    onnx_filename = f"{lang_country}-{voice}-{quality}.onnx"
+    config_filename = f"{onnx_filename}.json"
+
+    files = list_files(base)
+    onnx_size = next((f["size"] for f in files if f["path"].endswith(onnx_filename)), None)
+    if onnx_size is None:
+        return None
+
+    cfg_url = f"{HF_RESOLVE_BASE}/{base}/{config_filename}"
+    try:
+        cfg = fetch_json(cfg_url)
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)